### PR TITLE
Use std::make_unique whenever possible

### PR DESCRIPTION
**Pull request by mwestphal**
needs : #339

### CODE REVIEW

**Comment by Meakk** [comment]
FYI, this is covered by clang-tidy. See https://github.com/f3d-app/f3d/pull/329

**Comment by mwestphal** [comment]
> FYI, this is covered by clang-tidy. See #329

Ok, but I may merge this before your tidy work :)

File: library/public/engine.h [hunk 1]
```diff
@@ -25,6 +25,7 @@
 #include "window.h"
 
 #include <map>
+#include <memory>
```
Comment (Meakk) [inline library/public/engine.h:28]: to remove

File: library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx [hunk 2]
```diff
@@ -33,6 +33,8 @@
 #include <assimp/postprocess.h>
 #include <assimp/scene.h>
 
+#include <memory>
```
Comment (Meakk) [inline library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx:36]: is it required?

**Review by Meakk — CHANGES_REQUESTED** [review]

File: library/public/engine.h (hunk 1 above)
Comment (mwestphal) [reply library/public/engine.h:28]: no

File: library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx (hunk 2 above)
Comment (mwestphal) [reply library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx:36]: yes

**Review by Meakk — APPROVED** [review]

**Pull request merged**